dns/pfSense-pkg-bind9/files/usr/local/pkg/bind.inc

```diff
@@ -112,10 +112,13 @@ function bind_sync() {
 
 	global $config;
 	conf_mount_rw();
+	$bind = $config['installedpackages']['bind']['config'][0];
 	// Create rndc
 	$rndc_confgen = "/usr/local/sbin/rndc-confgen";
+	$rndc_port = $bind['controlport'] == "" ? "953" : $bind['controlport'];
+	$rndc_confgen_opts = "-p$rndc_port";
```

Comment:
For this style of reference, you should use braces around inline variables, for example this would be `"-p {$rndc_port}"`

However, since this is passing user input to a shell command, it must be escaped better, so do this instead:

    $rndc_confgen_opts = '-p ' . escapeshellarg($rndc_port);